src/main/webapp/i18n/de/settings.json

```
@@ -12,9 +12,9 @@
         "messages": {
             "error": {
                 "fail": "<strong>Es ist ein Fehler aufgetreten!</strong> Die Einstellungen konnten nicht gespeichert werden.",
-                "emailexists": "<strong>E-Mail wird bereits verwendet!</strong> Bitte wähle eine andere aus."
+                "emailexists": "<strong>E-Mail ist bereits in Verwendung!</strong> Bitte wähle eine andere aus."
             },
-            "success": "<strong>Einstellungen wurden gespeichert!</strong>",
+            "success": "<strong>Einstellungen sind gespeichert!</strong>",
```

```suggestion
            "success": "<strong>Einstellungen wurden gespeichert!</strong>",
```
Same as a few above, I'd focus on the fact the settings got saved and not so much on the fact saved settings exist